test/Serilog.Sinks.PeriodicBatching.Tests/PeriodicBatchingSinkTests.cs

```diff
@@ -109,4 +109,34 @@ public void ExecutionContextDoesNotFlowToBatchedSink()
 
         Assert.Equal(default(int), observed);
     }
+    
+    [Fact]
+    public async Task EagerlyEmitFirstEventShouldWriteBatchImmediately()
```

Comment:
Maybe change the name to be simular to the other tests

```suggestion
    public async Task WhenEagerlyEmitFirstEventIsTrueItShouldWriteToBatchImmediately()
```